test/parser/next/decorators.ts

```diff
@@ -363,5 +364,15 @@ describe('Next - Decorators', () => {
     ],
     [`class A { @dec\nx }`, Context.OptionsNext | Context.OptionsRanges | Context.OptionsLoc],
     [`class A { @dec\nx(){} }`, Context.OptionsNext | Context.OptionsRanges | Context.OptionsLoc],
+    ['@dec export class E {};', Context.OptionsNext | Context.Module | Context.OptionsRanges | Context.OptionsLoc],
+    [
+      '@dec export default class {};',
+      Context.OptionsNext | Context.Module | Context.OptionsRanges | Context.OptionsLoc,
+    ],
+    // ['export @dec class E {};', Context.OptionsNext | Context.Module | Context.OptionsRanges | Context.OptionsLoc],
```

Comment:
@3cp bug? We can't parse this.

Reply:
It's in spec.
> Class decorators may exclusively come before, or after, export/export default.

Let me know if you want to fix it here, or with a new PR.

Reply:
https://github.com/meriyah/meriyah/issues/454